### PR TITLE
Revert "Change Calls to FeeManager contract"" with Fallback

### LINKED ISSUE #165: Get fees from the FeesManager
Due to bytecode size limit in the new version of OB contracts the functionality to keep and calculate fees for specific tokens is moved to a separate contract [`FeeManager`](https://github.com/poanetwork/omnibridge/blob/master/contracts/upgradeable_contracts/modules/fee_manager/OmnibridgeFeeManager.sol). 

It means that the methods `HOME_TO_FOREIGN_FEE`, `FOREIGN_TO_HOME_FEE`, `calculateFee` and `isRewardAddress` are not available in the OB mediator contract anymore.

In order to use these methods, the OB fee manager contract address must be get from the mediator contract ([`feeManager()`](https://github.com/poanetwork/omnibridge/blob/ea0ffa6f015da024306d80f61bd271c4268b1f7a/contracts/upgradeable_contracts/modules/fee_manager/OmnibridgeFeeManagerConnector.sol#L33)) first. After that the methods listed above can be requested from the fee manager contract.

### PR DESCRIPTION
**Pull request by alexkeating**
Aims to close #165

### CODE REVIEW

File: packages/react-app/src/hooks/useMediatorInfo.js [hunk 1]
```diff
@@ -8,58 +8,62 @@ import { useEffect, useState } from 'react';
...
         abi,
         ethersProvider,
       );
 
       mediatorContract
-        .FOREIGN_TO_HOME_FEE()
-        .then(feeType => setForeignToHomeFeeType(feeType))
-        .catch(feeTypeError => logError({ feeTypeError }));
+        .getBridgeInterfacesVersion()
+        .then(version => {
```
Comment (dan13ram) [inline packages/react-app/src/hooks/useMediatorInfo.js:34]: can we remove these extra state variables for version and do the if check inside the `then` clause?

File: packages/react-app/src/hooks/useMediatorInfo.js [hunk 2]
```diff
@@ -8,58 +8,62 @@ import { useEffect, useState } from 'react';
...
 
-      mediatorContract
-        .HOME_TO_FOREIGN_FEE()
-        .then(feeType => setHomeToForeignFeeType(feeType))
-        .catch(feeTypeError => logError({ feeTypeError }));
+      if (interfaceMajorVersion >= 2 && interfaceMinorVersion >= 1) {
+        mediatorContract
+          .feeManager()
+          .then(feeManager => setFeeManagerAddress(feeManager))
+          .catch(feeManagerAddressError =>
```
Comment (dan13ram) [inline packages/react-app/src/hooks/useMediatorInfo.js:44]: `feeManager => setFeeManagerAddress(feeManager)` can be replaced with `setFeeManager`, removing creation of a new method

**Review by dan13ram — CHANGES_REQUESTED** [review]
Small change to reduce number of UI refreshes

**Comment by dan13ram** [comment]
also could you rebase with `develop` branch ?

File: packages/react-app/src/hooks/useMediatorInfo.js (hunk 1 above)
Comment (alexkeating) [reply packages/react-app/src/hooks/useMediatorInfo.js:34]: Can do!

File: packages/react-app/src/hooks/useMediatorInfo.js (hunk 2 above)
Comment (alexkeating) [reply packages/react-app/src/hooks/useMediatorInfo.js:44]: I don't see the setFeeManager method. I remember seeing the method when reverting, but I think I renamed it to setFeeManagerAddress as that is a more accurate name.

File: packages/react-app/src/hooks/useMediatorInfo.js [hunk 3]
```diff
@@ -8,58 +8,62 @@ import { useEffect, useState } from 'react';
...
+        .catch(bridgeVersionError => logError({ bridgeVersionError }));
 
-      mediatorContract
-        .HOME_TO_FOREIGN_FEE()
-        .then(feeType => setHomeToForeignFeeType(feeType))
-        .catch(feeTypeError => logError({ feeTypeError }));
+      if (interfaceMajorVersion >= 2 && interfaceMinorVersion >= 1) {
+        mediatorContract
+          .feeManager()
+          .then(feeManager => setFeeManagerAddress(feeManager))
```
Comment (dan13ram) [inline packages/react-app/src/hooks/useMediatorInfo.js]:
```suggestion
          .then(setFeeManagerAddress)
```
@alexkeating I'm sorry I typed that on my phone. This was my suggestion

File: packages/react-app/src/hooks/useMediatorInfo.js (hunk 2 above)
Comment (alexkeating) [reply packages/react-app/src/hooks/useMediatorInfo.js:36]: I did add a setFeeManager method in the latest commit

File: packages/react-app/src/hooks/useMediatorInfo.js (hunk 3 above)
Comment (alexkeating) [reply packages/react-app/src/hooks/useMediatorInfo.js]: Ah cool thanks!

**Review by dan13ram — APPROVED** [review]

**Pull request merged**